<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubjectDialog</class>
 <widget class="QDialog" name="SubjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>科目详情</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>科目名称(&amp;N):</string>
       </property>
       <property name="buddy">
        <cstring>nameLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>描述(&amp;D):</string>
       </property>
       <property name="buddy">
        <cstring>descriptionTextEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPlainTextEdit" name="descriptionTextEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="colorLabel">
       <property name="text">
        <string>颜色(&amp;C):</string>
       </property>
       <property name="buddy">
        <cstring>colorButton</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="colorButton">
         <property name="text">
          <string>选择颜色</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="colorPreviewLabel">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid black;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="tagsLabel">
       <property name="text">
        <string>标签(&amp;T):</string>
       </property>
       <property name="buddy">
        <cstring>tagsLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="tagsLineEdit">
       <property name="placeholderText">
        <string>逗号分隔, 例如: 重点, 复习</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
        <property name="text"><string/></property>
        <property name="alignment"><set>Qt::AlignCenter</set></property>
        <property name="wordWrap"><bool>true</bool></property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>descriptionTextEdit</tabstop>
  <tabstop>colorButton</tabstop>
  <tabstop>tagsLineEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>